v2/component/launcher.go

```diff
@@ -178,100 +182,125 @@ func NewLauncher(runtimeInfo string, options *LauncherOptions) (*Launcher, error
 	}, nil
 }
 
-func (l *Launcher) prepareInputs(ctx context.Context) error {
+func (l *Launcher) generateOutputURI(name string) string {
+	blobKey := path.Join(l.options.PipelineName, l.options.PipelineRunID, l.options.TaskName, name)
+	return l.bucketConfig.uriFromKey(blobKey)
+}
+
+func localPathForURI(uri string) (string, error) {
+	if strings.HasPrefix(uri, "gs://") {
+		return "/gcs/" + strings.TrimPrefix(uri, "gs://"), nil
```

Comment:
nit: shall we use `/tmp/gcs/` as prefix?

I remember there are people who tries to run KFP as non-root, `/tmp` folder is guaranteed to be accessible while `/gcs/` might not.

Reply:
I thought this was accommodating gcsfuse, which mounts to `/gcs/`.